otx/algorithms/classification/configs/efficientnet_b0_cls_incr/model_supcon.py

```diff
@@ -0,0 +1,26 @@
+"""EfficientNet-B0 for multi-class config."""
```

Comment:
This is optional, how about "EfficientNet-B0 config for multi-class with contrastive loss for smaller dataset"

Reply:
Yes, that sounds better, I will update the descriptions of all models :)